tools/stern/test/integration/API.hs

```diff
@@ -93,14 +95,34 @@ tests s =
       test s "GET /i/consent" testGetConsentLog,
       test s "GET /teams/:id" testGetTeamInfo,
       test s "GET i/user/meta-info?id=..." testGetUserMetaInfo,
-      test s "/teams/:tid/search-visibility" testSearchVisibility
+      test s "/teams/:tid/search-visibility" testSearchVisibility,
+      test s "i/oauth/clients" testCrudOAuthClient
```

Comment:
why the `/i`?  stern doesn't have any public-facing api, so the distinction doesn't really work here i think.

Reply:
I've seen it on other stern endpoints, too, so I adapted it. I guess, it just serves as a marker to make it obvious that this is calling an internal endpoint. Should I remove it?

Reply:
when i complained about this i thought this was the only end-point in stern with an `/i`, and that stern is only calling internal end-points.  both assumptions may be wrong.  use your own judgement, if it's consistent with what stern does everywhere, keep it.

Reply:
ah, i see i'm a bit late.  :)  either way it's not a big deal.